### ISSUE TITLE
iD should suggest removing noname=yes from roads with a name

### ISSUE THREAD

**Issue by tas50** [openstreetmap/iD#8306]
I've come across a number of named roads that still have the noname=yes tag set. This is invalid and should be marked so when editing.

<img width="373" alt="image" src="https://user-images.githubusercontent.com/1015200/104987213-ff369d80-59c9-11eb-8706-80e97fd5c969.png">

**Comment by 1ec5** [comment]
Yeah, there’s [a whole bunch](https://wiki.openstreetmap.org/wiki/SPARQL_examples#Features_tagged_with_mutually_exclusive_keys) of mutually exclusive keys that iD could warn about (or that there could be a MapRoulette challenge about). Sophox is currently down, but here’s the [list of keys](https://wiki.openstreetmap.org/wiki/Special:WhatLinksHere/Property:P44) to check.

Note that it may be the `name` that’s incorrect, so the suggestion should be to remove one or the other tag, not necessarily the `noname=yes`. (If the `name` is incorrect, perhaps `not:name` should be added.)

**Comment by k-yle** [comment]
This was fixed in #10035
